tests/unittests/imageTest.cpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "glow/Base/Image.h"
+
+#include "llvm/Support/FileSystem.h"
+
+#include "gtest/gtest.h"
+
+#include <cstdio>
+#include <utility>
+
+using namespace glow;
+
+TEST(Image, writePngImage) {
+  auto range = std::make_pair(0.f, 1.f);
+  Tensor localCopy;
+  bool loadSuccess = !readPngImage(
+      &localCopy, ROOT_BINARY_DIR "/tests/images/imagenet/cat_285.png", range);
```

Comment:
Up to us.
The problem with the onnxImporterTest is if you run it from outside of the build directory, it will fail.
E.g., ./tests/onnxImporterTest works, (cd tests; ./onnxImporterTest) doesn't.

I can remove the logic for the ROOT_BINARY_DIR stuff if that's fine for everybody.